controller/test-files/search/show/in_topology_A-B-C_and_A-D_search_for/A_with_tree-view=false.res.golden.json

```diff
@@ -0,0 +1,88 @@
+{
+  "data": [
+    {
+      "attributes": {
+        "system.created_at": "0001-01-01T00:00:00Z",
+        "system.number": 1,
+        "system.order": 1000,
+        "system.remote_item_id": null,
+        "system.state": "new",
+        "system.title": "A",
+        "system.updated_at": "0001-01-01T00:00:00Z",
+        "version": 0
+      },
+      "id": "00000000-0000-0000-0000-000000000001",
+      "links": {
+        "related": "http:///api/workitems/00000000-0000-0000-0000-000000000001",
```

Comment:
I think that `self` is the only link you should provide here

Reply:
Please see this followup issue for these JSONAPI compliance issues: https://github.com/fabric8-services/fabric8-wit/issues/1860